src/actions/types.js

```diff
@@ -0,0 +1,39 @@
+//Donations
```

Comment:
I feel like this file name should be action-types or in a directory called constants

Reply:
done src/actions/actionTypes.js